daphne/src/messages.rs

```diff
@@ -4,22 +4,30 @@
 //! Messages in the DAP protocol.
 
 use prio::codec::{
-    decode_u16_items, decode_u32_items, encode_u16_items, encode_u32_items, CodecError, Decode,
-    Encode,
+    decode_u16_items, decode_u24_items, decode_u32_items, decode_u8_items, encode_u16_items,
+    encode_u24_items, encode_u32_items, encode_u8_items, CodecError, Decode, Encode,
 };
 use serde::{Deserialize, Serialize};
 use std::{
     convert::{TryFrom, TryInto},
     fmt::Debug,
     io::{Cursor, Read},
+    str::from_utf8,
 };
+use url::Url;
 
 const KEM_ID_X25519_HKDF_SHA256: u16 = 0x0020;
 const KEM_ID_P256_HKDF_SHA256: u16 = 0x0010;
 const KDF_ID_HKDF_SHA256: u16 = 0x0001;
 const AEAD_ID_AES128GCM: u16 = 0x0001;
 const QUERY_TYPE_TIME_INTERVAL: u8 = 0x01;
 const QUERY_TYPE_FIXED_SIZE: u8 = 0x02;
+const PRIO3_AES128_COUNT: u32 = 0x00000000;
+const PRIO3_AES128_SUM: u32 = 0x00000001;
+const PRIO3_AES128_HISTOGRAM: u32 = 0x00000002;
+const POPLAR1_AES128: u32 = 0x00001000;
+const DP_MECHANISM_RESERVED: u8 = 0x00;
```

Comment:
I don't think we need to define this constant, since the reserved codepoint (0x00) will never be used for a real DpConfig. We could just handle this as DpConfig::NotImplemented, I suppose?

Reply:
I wasn't sure about that one.  The draft says it is "reserved for testing purposes" but I didn't know if that meant it would be instantiated for testing or not, so I gave it a definition.  Happy to remove it though!

Reply:
Fine to keep it I think :)